src/common/signalk/SKNMEA2000Parser.cpp

```diff
@@ -80,42 +128,207 @@ const SKUpdate& SKNMEA2000Parser::parse128259(const SKSourceInput& input, const
+      update->setTimestamp(timestamp);
+
+      SKSource source = SKSource::sourceForNMEA2000(input, msg.PGN, msg.Priority, msg.Source);
+      update->setSource(source);
+      // -> Current rudder angle, +ve is rudder to Starboard
+      update->setSteeringRudderAngle(rudderPosition);
+
+      _sku = update;
+      return *_sku;
+    }
+  }
+  DEBUG("Unable to parse NMEA2000 with PGN %i", msg.PGN);
+  return _invalidSku;
+}
+// *****************************************************************************
+//    PGN 127250 VESSEL HEADING RAPID
+//    Heading sensor value with a flag for True or Magnetic.
+//    Heading               Heading in radians
+//      - Deviation         Magnetic deviation in radians. Use N2kDoubleNA for undefined value.
+//      - Variation         Magnetic variation in radians. Use N2kDoubleNA for undefined value.
+//      tN2kHeadingReference
+//        N2khr_true=0,
+//        N2khr_magnetic=1
+// *****************************************************************************
+const SKUpdate& SKNMEA2000Parser::parse127250(const SKSourceInput& input, const tN2kMsg& msg, const SKTime& timestamp) {
+  unsigned char sid;
+  tN2kHeadingReference headingReference;
+  double heading = N2kDoubleNA;
+  double deviation = N2kDoubleNA;
+  double variation = N2kDoubleNA;
+
+  if (ParseN2kHeading(msg,sid,heading,deviation,variation,headingReference)) {
+    if (!N2kIsNA(heading) && heading >= 0 && heading <= 2 * M_PI) {
+      if (headingReference == N2khr_magnetic) {
+        //TODO put 3 when updated to deviation
+        SKUpdateStatic<2> *update = new SKUpdateStatic<2>();
+        update->setTimestamp(timestamp);
+
+        SKSource source = SKSource::sourceForNMEA2000(input, msg.PGN, msg.Priority, msg.Source);
+        update->setSource(source);
+        update->setNavigationHeadingMagnetic(heading);
+        if (!N2kIsNA(variation))
+          update->setNavigationMagneticVariation(variation);
+          /* coming when Signal K adds deviation
+          if (!N2kIsNA(deviation))
+            update->setNavigationMagneticDeviation(deviation);
+          */
+        _sku = update;
+        return *_sku;
+      }
+
+      if (headingReference == N2khr_true) {
+        SKUpdateStatic<1> *update = new SKUpdateStatic<1>();
+        update->setTimestamp(timestamp);
+
+        SKSource source = SKSource::sourceForNMEA2000(input, msg.PGN, msg.Priority, msg.Source);
+        update->setSource(source);
+        update->setNavigationHeadingTrue(heading);
+        _sku = update;
+        return *_sku;
+      }
+    }
+  }
+  DEBUG("Unable to parse NMEA2000 with PGN %i", msg.PGN);
+  return _invalidSku;
+}
+
+// *****************************************************************************
+//    PGN 128000 Nautical Leeway Angle (new 2017)
+// https://www.nmea.org/Assets/20170204%20nmea%202000%20leeway%20pgn%20final.pdf
+// Upcoming in Timos library:
+// void SetN2kPGN128000(tN2kMsg &N2kMsg, unsigned char SID, double Leeway) {
```

Comment:
Timo has merged this now if you want (or it can stay as a comment and wait for a future PR).

Reply:
Changed the code, but
´case  N2kWind_Apparent:´  is not working.
Timo has updated the library already and I made an update in Platformio too.
But I think due to platformio.ini it is called the old lib?

Reply:
In platformio.ini you need to change the commit number we are referencing to:
```
  https://github.com/sarfata/NMEA2000#feature/fix-build-for-mac
```
becomes:
```
  https://github.com/ttlappalainen/NMEA2000#0aea81d 
 ```

It will conflict because I already did it on develop once before Timo merged. I will do it for you.